test/subproviders/etherscan.js

```diff
@@ -1,4 +1,4 @@
-const sha3 = require('ethereumjs-util').sha3;
+const keccak256 = require('ethereumjs-util').keccak256;
```

Comment:
This is a nitpick, feel free to take or leave, but it's probably more idiomatic to say this, yeah?

``` suggestion
const { keccak256 } = require('ethereumjs-util')
```